backends/p4tools/modules/testgen/targets/bmv2/test_backend/protobuf.cpp

```diff
@@ -228,7 +200,7 @@ inja::json Protobuf::getControlPlaneForTable(const TableMatchMap &matches,
     return rulesJson;
 }
 
-inja::json Protobuf::getSend(const TestSpec *testSpec) {
```

Comment:
It's a bit odd to me that you're not also using the 'common' version here? It seems like it has to do with formatting, but perhaps that could come from another input to the function or something?

Same below.

Reply:
Yes I would like to use the common version, but the way hex is interpreted in the other frameworks (STF for some reason uses raw hex without a prefix) versus how it is interpreted in text protobuf (using text separators) seems incompatible. Maybe I can simplify this.

Reply:
I see. I could imagine using that as an input to the function somehow. Or having each subclass provide a 'InterpretHex' function of their own (e.g.).

I would ideally fix that in this CL, but I can approve it if you think it's better to do this as a second pass later.

Reply:
https://github.com/p4lang/p4c/issues/4230